Documentation/policy/language.rst

```diff
@@ -845,6 +850,12 @@ latter rule will have no effect.
 .. note:: Layer 7 rules are not currently supported in `HostPolicies`, i.e.,
           policies that use :ref:`NodeSelector`.
 
+.. note:: Layer 7 policies --and pod annotations-- result in traffic being
+   proxied through an Envoy container provided in each Cilium agent. As
```

Comment:
```suggestion
   proxied through an Envoy instance provided in each Cilium agent pod. As
```

Reply:
^^ Currently Envoy is just a process in the Cilium pod, but this could plausibly change at some point. Using `instance` here would cover both, so it won't be wrong if we forget to update this line later on :)